app/routes/wiki-page.js

```diff
@@ -148,7 +148,7 @@ export default Route.extend(
 				}
 			} else {
 				this.get('logger').warn('Unsupported page');
-				logEvent('Model is missing (unsupported page)');
+				logError('Model is missing (unsupported page)');
```

Comment:
Do we still need this logging? if yes let's create a ticket to remove it in the future.

Reply:
Seems like there are no events of this kind in the event-logger index in Kibana. I'll remove it.